src/BikramSambat.ts

```diff
@@ -22,4 +22,10 @@ export default class BikramSambat {
     this.month = month
     this.day = day
   }
+  public getPreviousMonth():number{
+    return this.month - 1
+  }
```

Comment:
we also need to account the case of first month, when month = 1, if you decrease it by 1 then it will be month zero which is invalid date in our case.
if date is 2080-01-23
then then previous month would be 2079-12-23
return should be `{index: 12, en: 'Chaitra', np: 'चैत्र', ad: 'Mar/Apr' }`
you can find `NepaliMonthsData` [here](https://github.com/askbuddie/nepali-date-picker/blob/77537082d4b192929347856fbc2de443a2513845/src/data/nepali-months.ts#L1)

Reply:
In case of zero , we should return last value . i.e Chaitra.

Reply:
`public getPreviousMonth():Month | null {
    if(this.month === 0){
      return NepaliMonthsData[NepaliDaysData.length - 1]
    }
    return NepaliMonthsData[this.month - 1]
  }`

Reply:
Is this correct

<img width="752" alt="Screenshot 2023-08-18 at 08 27 23" src="https://github.com/askbuddie/nepali-date-picker/assets/57742339/0ffdb85a-3832-49cf-a375-1efa1f7a2e30">

Reply:
**true**
to get month index you can do sth like
`const month = this.month===1 ? 12 : this.month-1`
then return the object from `NepaliMonthsData`
also dont forget to account the "invalid date" case like in `getNextMonth()`

Reply:
Something like this ?

<img width="851" alt="Screenshot 2023-08-18 at 09 54 56" src="https://github.com/askbuddie/nepali-date-picker/assets/57742339/707098c3-b68c-42e9-b168-c6cf29ede5ee">

Reply:
You dont need to take mod here, as there is no chance of getting more month value more than 12.
` const month = this.month===1 ? 12 : this.month-1`
this will give you the month index and you can just use that to access Month Data